controllers/lexemes.js

```diff
@@ -0,0 +1,86 @@
+const express = require('express');
+const router = express.Router();
+const Lexeme = require('../models').Lexeme;
+
+
+// "id" parameter –> params
+router.param( 'id', (req, res, next, id) => {
+  Lexeme.findById( req.params.id, (error, lexeme) => {
+    if ( error ) return next(error);
+    if ( !lexeme ) {
+      error = new Error( 'Not found' );
+      error.status = 404;
+      return next(error)
+    }
+    req.lexeme = lexeme;
+    return next();
+  });
+});
+
+
+// get all lexemes (for now), full data (–> editing)
+router.get( '/full', (req, res, next) => {
```

Comment:
If you don't use `/` for the next one, here would be a good fit otherwise `/all` sounds better 🤙